tests/rest/media/v1/test_url_preview.py

```diff
@@ -79,6 +79,9 @@ def make_homeserver(self, reactor, clock):
 
         config["media_storage_providers"] = [provider_config]
 
+        # Route the HTTP version to an HTTP endpoint so that the tests work.
```

Comment:
I don't understand this comment?

Reply:
This looks like it was my original comment, it was trying to avoid requiring HTTPS.

Reply:
ok, but I don't think it makes sense in its new context.